common/eventconsumption/eventfilter.cpp

```diff
@@ -97,12 +102,16 @@ class CEventFilter : public CInterfaceOf<IEventFilter>
             return true;
         }
         virtual bool acceptToken(const char* token, FilterTermComparison comp) = 0;
+    protected:
+        CEventConsumingOp& operation;
     };
 
     struct StringFilterTerm : public FilterTerm
     {
         StringMatchHelper helper;
 
+        using FilterTerm::FilterTerm;
```

Comment:
I'm curious why these are needed.

Reply:
Subclasses get automatic pass-through constructors matching the base class. Especially convenient when many subclasses are unaware of the base class parameters.

Reply:
nice.  I didn't know that...